api/server/routes_create.go

```diff
@@ -44,13 +43,13 @@ func (s *Server) handleRouteCreate(c *gin.Context) {
 		return
 	}
 
-	err = s.Runner.EnsureImageExists(ctx, &task.Config{
-		Image: wroute.Route.Image,
-	})
-	if err != nil {
-		c.JSON(http.StatusBadRequest, simpleError(models.ErrUsableImage))
-		return
-	}
+	// err = s.Runner.EnsureImageExists(ctx, &task.Config{
+	// 	Image: wroute.Route.Image,
+	// })
+	// if err != nil {
+	// 	c.JSON(http.StatusBadRequest, simpleError(models.ErrUsableImage))
+	// 	return
+	// }
```

Comment:
Ya, we could. I'm not sure if it would be used much though, it really slows down the call, seconds instead of milliseconds.